### PR TITLE
added plugin support for relative links and make fixes/updates to other .md files

### PR DESCRIPTION
**Pull request by kmoscoe**
This change adds support for a relative links plug-in (https://github.com/benbalter/jekyll-relative-links/blob/main/README.md), which otherwise don't work in Github Pages sites. 

In addition, made the following changes to the readme files:

* Added a link to Kramdown docs
* Fixed broken link from readme to contributing file
* Fixed incorrect link to Google dev style guide
* Added some background info on how the site is hosted and generated and navigation bar.
* Added some text to the style guide to explain how to do non-obvious things like linking and TOCs
* Added a tip for running the site locally with an alternate IP address
* Cleaned up the contributing instructions to not use hard-coded variables
* Edited to use actual Google dev style guide requirements like sentence case everywhere, uppercase, italicized variable names, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,50 +18,67 @@ to make public data accessible to everyone.
...
+<pre>
 git add .
-git commit -m "commit message"
-git push -u origin new_branch_name
-```
+git commit -m "<var>COMMIT_MESSAGE<var>"
+git push -u origin <var>BRANCH_NAME</var>
+</pre>
+
+Then, in guthub.com, in your forked repo, you can send a pull request. You will need to assign at least one reviewer to approve.
```
Comment (kmoscoe) [inline README.md:74]:
```suggestion
Then, in github.com, in your forked repo, you can send a pull request. You will need to assign at least one reviewer to approve.
```

File: Gemfile [hunk 2]
```diff
@@ -18,4 +18,8 @@ group :jekyll_plugins do
   gem "jekyll-tabs"
 end
 
+group :jekyll_plugins do  
+  gem "jekyll-relative-links"
```
Comment (beets) [inline Gemfile:22]: You should be able to add this to the block in line 17

File: README.md [hunk 3]
```diff
@@ -18,50 +18,71 @@ to make public data accessible to everyone.
...
-Clone your forked repo to your desktop.
+    This adds your fork as the remote called `origin`.
 
-Add datacommonsorg/docsite repo as a remote:
+1. Add `datacommonsorg/docsite` repo as a remote:
 
-```shell
-git remote add dc https://github.com/datacommonsorg/docsite.git
-```
+    <pre>
```
Comment (beets) [inline README.md:56]: Any reason why the backticks don't work as well here? Is it to get the var formatting?

**Review by beets — COMMENTED** [review]
Thanks for these updates!

Could you also please add more context to the PR description?

File: Gemfile (hunk 2 above)
Comment (kmoscoe) [reply Gemfile:22]: Done!

File: README.md (hunk 3 above)
Comment (kmoscoe) [reply README.md:56]: Yep. Google Dev style guide requires italics for variables. Markdown uses _ which gets rendered literally if you use the backticks. It's kind of a pain, but that's what we have.

**Comment by beets** [comment]
thanks for the update. i think you need to push your changes to your remote branch.

for this and future PR's, please add the "why" for the change to the PR description.

**Comment by kmoscoe** [comment]
Actually, there's a couple of other changes I need to make. Is there a way
to add more info to the PR description? I couldn't figure it out.

**Comment by kmoscoe** [comment]
Like, for example, in Critique, you have a CL title and a CL description; I
can't figure out how to add a longer description.

**Review by beets — APPROVED** [review]
thanks for the fixes!

**Pull request merged**